DistributionPackages/Neos.NeosIo.ServiceOfferings/Resources/Private/Fusion/Components/Provider.ShowCases.fusion

```diff
@@ -0,0 +1,16 @@
+prototype(Neos.NeosIo.ServiceOfferings:Component.Provider.ShowCases) < prototype(Neos.Fusion:Component) {
+    showCases = null
+
+    @if.hasShowCases = ${this.showCases}
+
+    renderer = afx`
+        <section class="service-provider__showcases">
+            <h2>Showcases</h2>
+            <div class="references__list g">
```

Comment:
What does the class `g`?

Reply:
The grid flex thingy from the old css framework